### PR TITLE
gpu: Add build targets for GPU rootfs initrd/image

### PR DESCRIPTION
**Pull request by zvonkok**
Preparation for complete GPU rootfs build step num 1 of N

### CODE REVIEW

File: tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh [hunk 1]
```diff
@@ -414,6 +414,39 @@ install_initrd_mariner() {
 	install_initrd "mariner"
 }
 
+#Instal NVIDIA GPU image
+install_image_nvidia_gpu() {
+	export AGENT_POLICY="yes"
+	export AGENT_INIT="yes"
```
Comment (stevenhorsman) [inline tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh:420]: @fidencio and/or @danmihai1 can probably confirm, but with the regorus crate and follow on changes I'm not sure if `AGENT_POLICY="yes"` (as it defaults to `yes` now) and `AGENT_INIT="yes"` are strictly required to use policy, but equally I don't think it's and issue to have them set as a double measure.

Comment (zvonkok) [reply tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh:420]: It is more to amplify and show that be it image or initrd we're always using AGENT_INIT and AGENT_POLICY I think it does not "hurt".

**Review by stevenhorsman — APPROVED** [review]
Looks okay to me.

**Review by beraldoleal — APPROVED** [review]
lgtm, thanks

**Pull request merged**